.goreleaser.yml

```diff
@@ -3,7 +3,10 @@
 before:
   hooks:
     # you may remove this if you don't use vgo
-    - go mod download
+    # Note: comment out because executed by travis ci "install" task
+    # - go mod download
```

Comment:
I think if `go mod download` is exected here then some git differences are generated.(e.g. go.mod, go.sum, README.md)